### PR TITLE
Page api/sfc-spec

### PR DESCRIPTION
**Pull request by imblackline**

### CODE REVIEW

File: src/api/sfc-spec.md [hunk 1]
```diff
@@ -32,63 +32,65 @@ export default {
...
+- [استفاده از vite-plugin-vue-gql با بلوک سفارشی: `<gql>`](https://github.com/wheatjs/vite-plugin-vue-gql)
+- [استفاده از vue-i18n با بلوک سفارشی: `<i18n>`](https://github.com/intlify/bundle-tools/tree/main/packages/vite-plugin-vue-i18n#i18n-custom-block)
 
-- [Gridsome: `<page-query>`](https://gridsome.org/docs/querying-data/)
-- [vite-plugin-vue-gql: `<gql>`](https://github.com/wheatjs/vite-plugin-vue-gql)
-- [vue-i18n: `<i18n>`](https://github.com/intlify/bundle-tools/tree/main/packages/vite-plugin-vue-i18n#i18n-custom-block)
+مدیریت بلوک های سفارشی به ابزارسازی بستگی دارد - اگر می خواهید ادغامی از بلوک های سفارشی خود را بسازید، برای اطلاعات بیشتر به [بخش ابزار ادغام بلوک های سفارشی کامپوننت های تک فایلی | SFC](/guide/scaling-up/tooling#sfc-custom-block-integrations) مراجعه کنید.
 
-Handling of Custom Blocks will depend on tooling - if you want to build your own custom block integrations, see the [SFC custom block integrations tooling section](/guide/scaling-up/tooling#sfc-custom-block-integrations) for more details.
+## استنتاج خودکار نام {#automatic-name-inference}
```
Comment (mostafa-nematpour) [inline src/api/sfc-spec.md:75]:
```suggestion
## پیش‌بینی خودکار نام {#automatic-name-inference}
```

File: src/api/sfc-spec.md [hunk 2]
```diff
@@ -32,63 +32,65 @@ export default {
...
 
-## Automatic Name Inference {#automatic-name-inference}
+یک SFC به طور خودکار نام مؤلفه را از **نام فایل** آن در موارد زیر استنتاج می کند:
 
-An SFC automatically infers the component's name from its **filename** in the following cases:
+- قالب بندی هشدار توسعه (Dev warning formatting)
+- بازرسی DevTools یا (DevTools inspection)
+- ارجاع بازگشتی به خود، به عنوان مثال فایلی به نام `FooBar.vue` می‌تواند در قالب خود به `<FooBar/>` اشاره کند. این اولویت کمتری نسبت به کامپوننت های صریحا ثبت‌شده/ایمپورت شده دارد.
```
Comment (mostafa-nematpour) [inline src/api/sfc-spec.md:81]:
```suggestion
- ارجاع بازگشتی به خود، به عنوان مثال فایلی به نام `FooBar.vue` می‌تواند در تمپلیت خود به `<FooBar/>` اشاره کند. این اولویت کمتری نسبت به کامپوننت های صریحا ثبت‌شده/ایمپورت شده دارد.
```

Comment (mostafa-nematpour) [inline src/api/sfc-spec.md:81]:
```suggestion
- ارجاع بازگشتی به خود، به عنوان مثال فایلی به نام `FooBar.vue` می‌تواند در تمپلیت خود به `<FooBar/>` اشاره کند. این اولویت کمتری نسبت به کامپوننت های صریحا ثبت‌شده/ایمپورت شده دارد.
```

**Review by mostafa-nematpour — COMMENTED** [review]
Thank you for your contribution.
Review the changes and accept them.


You can use this to fix `*.vue` direction:
Screenshot_2023-12-10_15_06_48

**Comment by imblackline** [comment]
Thank you for your corrections 🙏🏻
I fixed them in a new commit 
@mostafa-nematpour

**Comment by mostafa-nematpour** [comment]
Thank you.

**Pull request merged**